community/pmc-tool/README.md

```diff
@@ -29,3 +29,35 @@ Important
   Verify the correct installation of dependencies and the availability of the required modules before running the code.
   Make sure the token.txt file contains a valid access token for the GitHub API.
   If everything is set up correctly, the code will fetch and store pull request data in the SQLite database. Then, it will print information about each author and their corresponding pull request count to the console.
+
+
+ # Running the Django Application
+Follow these instructions to run the Django application:
+
+## Prerequisites
+Before running the application, make sure you have the following components installed:
+
+Python (version 3.7 or above)
+Django (version 3.0 or above)
+
+## Installing Dependencies
+
+1. Open the terminal and navigate to the root directory of the project.
+2. Install the required dependencies by running the following command:
+
+     pip install -r requirements.txt
+
+## Configuring the Database
+1. In the settings.py file, ensure that the database settings match your configuration. By default, SQLite is used as the database.
+2. Run the migrations to create the necessary database tables:
```

Comment:
can you add details about which database tables are created? (pull_requests, issues? others?)